examples/platform/k32w/app/ldscripts/chip-k32w061-linker.ld

```diff
@@ -328,5 +320,6 @@ SECTIONS
     PROVIDE(Chip_LOWPOWER_ChipSoftwareReset = 0x03003fa1);
 
     __StackLimit = _vStackTop - STACK_SIZE;
-    ASSERT(__StackLimit >= _end_boot_resume_stack, "Possible stack corruption with data/bss/boot_stack")
+    /* ASSERT(__StackLimit >= _end_boot_resume_stack, "Possible stack corruption with data/bss/boot_stack") */
```

Comment:
nit: remove commented code?

Reply:
thanks, @LuDuda . I refactored this code a little.